.gitignore

```diff
@@ -1,3 +1,10 @@
+*~
+/.vscode
+myvenv
```

Comment:
@quratulain25 For a good overview or a better gitignore you should visit https://github.com/github/gitignore